src/reconcile/rec_write.c

```diff
@@ -844,17 +844,18 @@ __rec_split_chunk_init(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_CHUNK *
     /* Don't touch the key item memory, that memory is reused. */
     chunk->key.size = 0;
     chunk->entries = 0;
-    __wt_rec_addr_ts_init(r, &chunk->start_durable_ts, &chunk->oldest_start_ts,
-      &chunk->oldest_start_txn, &chunk->stop_durable_ts, &chunk->newest_stop_ts,
-      &chunk->newest_stop_txn);
+    __wt_rec_addr_ts_init(r, &chunk->newest_start_durable_ts, &chunk->oldest_start_ts,
+      &chunk->oldest_start_txn, &chunk->newest_stop_durable_ts, &chunk->newest_stop_ts,
+      &chunk->newest_stop_txn, &chunk->prepare);
 
     chunk->min_recno = WT_RECNO_OOB;
     /* Don't touch the key item memory, that memory is reused. */
     chunk->min_key.size = 0;
     chunk->min_entries = 0;
-    __wt_rec_addr_ts_init(r, &chunk->min_start_durable_ts, &chunk->min_oldest_start_ts,
-      &chunk->min_oldest_start_txn, &chunk->min_stop_durable_ts, &chunk->min_newest_stop_ts,
-      &chunk->min_newest_stop_txn);
+    /* FIXME-prepare-support: chunk->prepare should be some sort of aggregated prepare? */
```

Comment:
The chunk prepare flag is updated when the aggregated timestamp is calculated in function `__wt_rec_addr_ts_update`. Because of this reason, I removed it.  Currently, this aggregated information is not saved in the checkpoint. In the next ticket, we will work on it.